docs/contributing.md

```diff
@@ -1,24 +1,32 @@
 # Contributing to FLORAS
 
-If you want to modify FLORAS or contribute, you can also install it directly from source. We are using pdm to manage the dependencies.
+If you want to modify FLORAS or contribute, you can install it directly from source. We are using pdm to manage the dependencies.
 ```
 pip install pdm
-git clone https://github.com/tulip-control/flowsynth
+git clone https://github.com/tulip-control/floras.git
 ```
-Navigate to the repo and run to install the FLORAS and all required dependencies:
+Navigate to the repo to install floras and all required dependencies:
 ```
+cd floras
 pdm install
 ```
-Next, install spot by running
+Next, install spot by running:
 ```
 pdm run python get_spot.py
 ```
+If you are using conda, instead of the above command, you can install spot directly from conda-forge (this is faster):
```

Comment:
```suggestion
If you are using [conda](https://conda.org/), instead of the above command, you can install spot directly from [conda-forge](https://conda-forge.org/) (this is faster):
```